Assignment2/solrac1999-JSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+	"@context": {
+  	"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "prefix": "http://example.org/assignment2b/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "fullname": "rdfs:label",
+    "hometown": "rdfs:label",
+    "age": "rdfs:label",
+    "university":"rdfs:label",
+    "parents":"rdfs:label",
+    "mother":"rdfs:label",
+    "father":"rdfs:label",
+    "subjects":"rdfs:label",
+    "subject":"rdfs:label",
+    "teachers":"rdfs:label",
+    "teacher":"rdfs:label",
+    "description":"rdfs:comment"
+  },
+  "@id": "prefix:solrac1999",
+  "@type": "prefix:",
+  "fullname": {"label":"Nombre", "@value":"Carlos Jimenez-Laiglesia Perez"},
+	"hometown":{"@value":"Madrid"},
+  "age":{"@value":20,"@type":"xsd:integer"},
+  "university": {"@value":"UPM"},
+  "parents": {
+		"mother": {"@value":"Pilar Perez"},
+		"father": {"@value":"Carlos Jimenez-Laiglesia"}
+	},
+	"subjects": {
+		"subject": [{

```

Comment:
You don't need it because it creates another blank node, thus is better to remove it and define "subjects" as an array -> "subjects": [ all the content thay you have into the "subject"]